WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -4,13 +4,6 @@ import WordPressComAnalytics
 
 
 
-/// These notifications are sent when the user Registers / Unregisters for Push Notifications.
-///
-public let NotificationsManagerDidRegisterDeviceToken   = "NotificationsManagerDidRegisterDeviceToken"
-public let NotificationsManagerDidUnregisterDeviceToken = "NotificationsManagerDidUnregisterDeviceToken"
```

Comment:
I guess we no longer have any observers for these notifications? How were they used previously? Just curious.